app/cni/pkg/cni/annotations_linux.go

```diff
@@ -36,6 +38,8 @@ var annotationRegistry = map[string]*annotationParam{
 	"iptablesLogs":                {"traffic.kuma.io/iptables-logs", "false", alwaysValidFunc},
 	"excludeInboundIPs":           {"traffic.kuma.io/exclude-inbound-ips", "", validateIPs},
 	"excludeOutboundIPs":          {"traffic.kuma.io/exclude-outbound-ips", "", validateIPs},
+	"virtualProbesEnabled":        {"kuma.io/virtual-probes", defaultVirtualProbeEnabled, alwaysValidFunc},
+	"virtualProbesPort":           {"kuma.io/virtual-probes-port", defaultVirtualProbePorts, validateSinglePort},
```

Comment:
Do we need both enabled and a port? Can't we just set the port to 0 to disable?

Reply:
These are existing annotations and I'm only introducing the support into `kuma-cni`. If we want to merge them, I think it's better to use a separated PR.

Reply:
virtualProxyProbesPort